packages/ia-components/sandbox/language-select/CloseMenu.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+
+/**
+ * Component that alerts if you click outside of it
+ */
+export default class CloseMenu extends Component {
+  constructor(props) {
+    super(props);
+
+    this.setWrapperRef = this.setWrapperRef.bind(this);
```

Comment:
Please bind these methods using `forEach` as we do in other component constructors like https://github.com/internetarchive/iaux/blob/master/packages/ia-components/sandbox/language-select/language-select.js#L80